lib/url.c

```diff
@@ -2027,7 +2027,7 @@ static CURLcode parseurlandfillconn(struct Curl_easy *data,
       return CURLE_OUT_OF_MEMORY;
   }
   else if(strlen(data->state.up.hostname) > 0xffff) {
-    failf(data, "Too long host name");
+    failf(data, "Too long host name (maximum is 65535)");
```

Comment:
maybe make a `#define` for it and use it in both places?

Reply:
sounds good.